sdk/documentintelligence/Azure.AI.DocumentIntelligence/README.md

```diff
@@ -14,8 +14,6 @@ Azure AI Document Intelligence is a cloud service that uses machine learning to
 
```

Comment:
Should we add about batch analysis in the readme too?

Reply:
We don't have a batch analysis sample yet. I intend to work on it while the release goes out. That said, I see it more like an advanced scenario, so I wouldn't include it in the README, just in samples.